packages/ListBox/test/specs/single/ListBox.single.spec.js

```diff
@@ -141,9 +141,34 @@ describe("Listbox single select", () => {
     }, 350);
   });
 
+  it.only("calls renderTrigger and changes the render method for label", () => {
```

Comment:
Should get rid of `.only`

Reply:
🤦‍♂

Reply:
https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-focused-tests.md and you never have to worry about this 😄